logic/user/user.go

```
@@ -49,6 +58,35 @@
+		)
+		messages := &request.RequestMessageSendMarkdown{
+			RequestMessageSend: request.RequestMessageSend{
+				ToUser:  *user.Username,
+				MsgType: "markdown",
+				AgentID: config.Config.Wechat.Work.Jdy.Id,
+			},
+			Markdown: &request.RequestMarkdown{
+				Content: content,
+			},
+		}
+		wxwork.Message.SendMarkdown(ctx, messages)
```

golangci-lint

Error return value of `wxwork.Message.SendMarkdown` is not checked

(errcheck)